include/cpeabs.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __CPEABS_H__
+#define __CPEABS_H__
+
+#include <stdint.h>
+
+#ifdef WEBCONFIG_BIN_SUPPORT
+
+#include <wdmp-c.h>
+#include <cimplog.h>
+#include <rbus/rbus.h>
+#include <rbus/rbus_object.h>
+#include <rbus/rbus_property.h>
+#include <rbus/rbus_value.h>
+#include <rbus-core/rbus_core.h>
+#include <rbus-core/rbus_session_mgr.h>
+
+#endif
+/*----------------------------------------------------------------------------*/
+/*                                   Macros                                   */
+/*----------------------------------------------------------------------------*/
+
+#ifdef WEBCONFIG_BIN_SUPPORT
```

Comment:
Do we need WEBCONFIG_BIN_SUPPORT check here? these are generic functions can be used for pod and local pc also right?